<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="testriq" failures="0" tests="19" name="testcases.TC_02_Add_To_Cart" time="110.403" errors="4" timestamp="2025-01-25T17:53:48 IST" skipped="12">
  <testcase classname="testcases.TC_02_Add_To_Cart" name="configuration" time="20.187">
    <error message="Appium HTTP server is not listening at http://127.0.0.1:4725/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4725, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m&#039;127.0.0.1&#039;[39m, port: [33m4725[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.665s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It&#039;s possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4725
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4725
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
" type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4725/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4725, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m, port: [33m4725[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.665s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4725
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4725
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)
at AndroidUtils.AppiumUtils.startAppiumServer(AppiumUtils.java:66)
at testcases.BaseTest.startAppiumServerAndInitializeDriver(BaseTest.java:20)
at testcases.TC_02_Add_To_Cart.configuration(TC_02_Add_To_Cart.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)
... 21 more
Caused by: java.net.ConnectException: Connection refused: getsockopt
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
at java.base/java.net.Socket.connect(Socket.java:751)
at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:531)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:636)
at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:280)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)
at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1319)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1252)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1138)
at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1067)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)
... 23 more
]]>
    </error>
  </testcase> <!-- configuration -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="configuration" time="20.102">
    <error message="Appium HTTP server is not listening at http://127.0.0.1:4727/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4727, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m&#039;127.0.0.1&#039;[39m, port: [33m4727[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.645s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It&#039;s possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4727
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4727
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
" type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4727/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4727, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m, port: [33m4727[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.645s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4727
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4727
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)
at AndroidUtils.AppiumUtils.startAppiumServer(AppiumUtils.java:66)
at testcases.BaseTest.startAppiumServerAndInitializeDriver(BaseTest.java:20)
at testcases.TC_02_Add_To_Cart.configuration(TC_02_Add_To_Cart.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)
... 21 more
Caused by: java.net.ConnectException: Connection refused: getsockopt
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
at java.base/java.net.Socket.connect(Socket.java:751)
at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:531)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:636)
at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:280)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)
at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1319)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1252)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1138)
at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1067)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)
... 23 more
]]>
    </error>
  </testcase> <!-- configuration -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="configuration" time="5.896">
    <error message="Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Device 94CAY0MBUV was not in the list of connected devices 
Host info: host: &#039;TESTRIQ&#039;, ip: &#039;192.168.29.64&#039;
Build info: version: &#039;4.28.1&#039;, revision: &#039;73f5ad48a2&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.5&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{platformName=ANDROID, appium:autoGrantPermissions=true, appium:udid=94CAY0MBUV, appium:appPackage=com.saucelabs.mydemoapp.android, appium:automationName=UiAutomator2, appium:systemPort=8300, appium:deviceName=94CAY0MBUV, appium:appActivity=com.saucelabs.mydemoapp.android.view.activities.SplashActivity, appium:app=G:\Automation Projects\testriq\src\main\java\apps\mda-2.2.0-25.apk}], desiredCapabilities=Capabilities {appium:app: G:\Automation Projects\test..., appium:appActivity: com.saucelabs.mydemoapp.and..., appium:appPackage: com.saucelabs.mydemoapp.and..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: 94CAY0MBUV, appium:systemPort: 8300, appium:udid: 94CAY0MBUV, platformName: ANDROID}}]
Capabilities {appium:app: G:\Automation Projects\test..., appium:appActivity: com.saucelabs.mydemoapp.and..., appium:appPackage: com.saucelabs.mydemoapp.and..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: 94CAY0MBUV, appium:systemPort: 8300, appium:udid: 94CAY0MBUV, platformName: ANDROID}" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Device 94CAY0MBUV was not in the list of connected devices 
Host info: host: 'TESTRIQ', ip: '192.168.29.64'
Build info: version: '4.28.1', revision: '73f5ad48a2'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{platformName=ANDROID, appium:autoGrantPermissions=true, appium:udid=94CAY0MBUV, appium:appPackage=com.saucelabs.mydemoapp.android, appium:automationName=UiAutomator2, appium:systemPort=8300, appium:deviceName=94CAY0MBUV, appium:appActivity=com.saucelabs.mydemoapp.android.view.activities.SplashActivity, appium:app=G:\Automation Projects\testriq\src\main\java\apps\mda-2.2.0-25.apk}], desiredCapabilities=Capabilities {appium:app: G:\Automation Projects\test..., appium:appActivity: com.saucelabs.mydemoapp.and..., appium:appPackage: com.saucelabs.mydemoapp.and..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: 94CAY0MBUV, appium:systemPort: 8300, appium:udid: 94CAY0MBUV, platformName: ANDROID}}]
Capabilities {appium:app: G:\Automation Projects\test..., appium:appActivity: com.saucelabs.mydemoapp.and..., appium:appPackage: com.saucelabs.mydemoapp.and..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: 94CAY0MBUV, appium:systemPort: 8300, appium:udid: 94CAY0MBUV, platformName: ANDROID}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:116)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:96)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:176)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:237)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:268)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:90)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:102)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)
at testcases.BaseTest.startAppiumServerAndInitializeDriver(BaseTest.java:35)
at testcases.TC_02_Add_To_Cart.configuration(TC_02_Add_To_Cart.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- configuration -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="configuration" time="20.058">
    <error message="Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m&#039;127.0.0.1&#039;[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.684s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It&#039;s possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
" type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\Testriq\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.5 (REV 4eddc999c5d9b705e7356152d2d5953a0dda3720)
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\Testriq\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\Testriq\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 1.684s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1904:16)
    at listenInCluster (node:net:1961:12)
    at doListen (node:net:2135:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)
at AndroidUtils.AppiumUtils.startAppiumServer(AppiumUtils.java:66)
at testcases.BaseTest.startAppiumServerAndInitializeDriver(BaseTest.java:20)
at testcases.TC_02_Add_To_Cart.configuration(TC_02_Add_To_Cart.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)
... 21 more
Caused by: java.net.ConnectException: Connection refused: getsockopt
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:682)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:592)
at java.base/java.net.Socket.connect(Socket.java:751)
at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:531)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:636)
at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:280)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)
at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1319)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1252)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1138)
at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1067)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)
... 23 more
]]>
    </error>
  </testcase> <!-- configuration -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithLoginAddToCartAndCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- WithLoginAddToCartAndCheckout -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithLoginAddToCartAndCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- WithLoginAddToCartAndCheckout -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithLoginAddToCartAndCheckout" time="17.012"/>
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithLoginAddToCartAndCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- WithLoginAddToCartAndCheckout -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithLoginAddToCartAndCheckout" time="0.000">
    <skipped/>
  </testcase> <!-- WithLoginAddToCartAndCheckout -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithoutLoginAddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- WithoutLoginAddToCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithoutLoginAddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- WithoutLoginAddToCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithoutLoginAddToCart" time="6.029"/>
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithoutLoginAddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- WithoutLoginAddToCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="WithoutLoginAddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- WithoutLoginAddToCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="RemoveProductFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveProductFromCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="RemoveProductFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveProductFromCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="RemoveProductFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveProductFromCart -->
  <testcase classname="testcases.TC_02_Add_To_Cart" name="RemoveProductFromCart" time="21.119"/>
  <testcase classname="testcases.TC_02_Add_To_Cart" name="RemoveProductFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveProductFromCart -->
</testsuite> <!-- testcases.TC_02_Add_To_Cart -->
